src/ui/views/menu.coffee

```diff
@@ -240,6 +240,13 @@ templateView = (viewstate) ->
             enabled: viewstate.loggedin
             click: (it) -> action 'convertemoji', it.checked
         }
+        {
+            type: 'checkbox'
+            label: i18n.__('menu.view.commands:Enable Commands')
```

Comment:
Can you add `menu.view.commands:Enable Commands` to `src/locale/en.json.template`

It is not used for application itself, but as a template for all translations.